File: PRACTICA DE CAMPO 01-S02/C#/S2_PRACTICACAMPO/s2_menu_caluladora.cs
double resultado=0;
Console.WriteLine("Ingrese el primer número:");
double numero1= double.Parse(Console.ReadLine());
Console.WriteLine("Ingrese el segundo número:");
double numero2= double.Parse(Console.ReadLine());
Console.WriteLine("Escriba número del Operador para el calculo(1-Suma, 2-Resta, 3-Multiplica, 4-Divide");
int operador= int.Parse(Console.ReadLine());
switch (operador)
{
    case 1:
        resultado = numero1 + numero2;
        Console.WriteLine("La Suma de los numero es: " + resultado);
        break;
    case 2:
        resultado = numero1 - numero2;
        Console.WriteLine("La Resta de los numero es: " + resultado);
        break;
    case 3:
        resultado = numero1 * numero2;
        Console.WriteLine("La Multiplicación de los numero es: " + resultado);
        break;
    case 4:
        if (numero2 == 0)        {
            Console.WriteLine("Divisor no puede ser igual a 0.");
        }else{
            resultado = numero1 / numero2;
            Console.WriteLine("La División de los numero es: " + resultado);
        }
        break;
    default:
        Console.WriteLine("Operador solo se permiten (1-Suma, 2-Resta, 3-Multiplica, 4-Divide).");
        break;
}

File: PRACTICA DE CAMPO 01-S02/PYTHON/s2_menu_caluladora.py
numero1=float(input("Ingrese Primer Número:"))
numero2=float(input("Ingrese Segundo Número:"))
operador=int(input("Escriba número del Operador para el calculo(1-Suma, 2-Resta, 3-Multiplica, 4-Divide):"))

match operador:
    case 1:
        resultado=float(numero1+numero2)
        print("La Suma de los numero es: ", resultado)
    case 2:
        resultado=float(numero1-numero2)
        print("La Resta de los numero es: ", resultado)
    case 3:
        resultado=float(numero1*numero2)
        print("La Multiplicación de los numero es: ", resultado)
    case 4:
        if numero2==0:
            print("Divisor no puede ser igual a 0.")
        else:
            resultado=float(numero1/numero2)
            print("La División de los numero es: ", resultado)
    case _:
        print("Operador solo se permiten (1-Suma, 2-Resta, 3-Multiplica, 4-Divide).")